backend/cfg/linear_to_cfg.ml

```diff
@@ -454,8 +454,8 @@ let to_basic (mop : Mach.operation) : C.basic =
 
 (** [traps] represents the trap stack, with head being the top. [trap_depths]
     is the depth of the trap stack. *)
-let rec create_blocks t (i : L.instruction) (block : C.basic_block)
-    ~trap_depth ~traps =
+let rec create_blocks (t : t) (i : L.instruction) (block : C.basic_block)
+    ~tailrec_label ~trap_depth ~traps =
```

Comment:
`~tailrec_label` does not change during construction of the `cfg` in `create_blocks`, so it would be better to have `tailrec_label` as a field of `Linear_to_cfg.t`.